packages/office-ui-fabric-react/src/components/Callout/examples/Callout.Directional.Example.tsx

```diff
@@ -39,25 +39,43 @@ export class CalloutDirectionalExample extends React.Component<any, ICalloutDire
 
     this.state = {
       isCalloutVisible: false,
-      isBeakVisible: true,
+      linkType: CalloutLinkType.beak,
       directionalHint: DirectionalHint.bottomLeftEdge
     };
   }
 
   public render() {
-    let { isCalloutVisible, isBeakVisible, directionalHint, gapSpace, beakWidth } = this.state;
+    let { isCalloutVisible, linkType, directionalHint, gapSpace, beakWidth } = this.state;
     //  ms-Callout-smallbeak is used in this directional example to reflect all the positions. Large beak will disable some position to avoid beak over the callout edge.
     return (
       <div className='ms-CalloutExample'>
         <div className='ms-CalloutExample-configArea'>
-          <Checkbox label='Show beak' checked={ isBeakVisible } onChange={ this._onShowBeakChange } />
+          <ChoiceGroup
+            label='Link type'
+            selectedKey={ linkType.toString() }
+            options={ [
+              {
+                key: CalloutLinkType.none.toString(),
```

Comment:
we shouldn't be using a key to store useful information. this value should only be used internally inside of react.